crates/data_components/src/postgres.rs

```diff
@@ -65,7 +65,7 @@ pub enum Error {
         source: db_connection_pool::dbconnection::GenericError,
     },
 
-    #[snafu(display("Unable to create Postgres connection pool: {source}"))]
+    #[snafu(display("{source}"))]
```

Comment:
Let's use the Snafu `transparent` attribute instead of this - its more idiomatic: https://docs.rs/snafu/latest/snafu/derive.Snafu.html#delegating-to-the-underlying-error

Reply:
Ok, I changed my mind - no more `transparent` :)

Reply:
The context information is what makes Snafu so great, so if we lose that - then its not worth it.